File: fix.py
with open('app/Services/Signia/Engines/QpseEngine.php', 'r', encoding='utf-8') as f:
    c = f.read()

c = c.replace(
    ' = round(([\'unit_value\'] * [\'quantity\']) + ([\'total_igv\'] ?? 0), 2);',
    ' = round([\'unit_value\'] * [\'quantity\'], 2);'
)
c = c.replace('abs([\'total_value\'] - )', 'abs([\'total_value\'] - )')
c = c.replace('(esperado: {})', '(esperado: {})')
c = c.replace(' += [\'total_value\'];', ' += [\'total_value\'] + ([\'total_igv\'] ?? 0);')

with open('app/Services/Signia/Engines/QpseEngine.php', 'w', encoding='utf-8') as f:
    f.write(c)

File: fix.php
<?php
$content = file_get_contents('app/Services/Signia/Engines/QpseEngine.php');
$content = str_replace(
    '$expectedTotal = round(($item[\'unit_value\'] * $item[\'quantity\']) + ($item[\'total_igv\'] ?? 0), 2);',
    '$expectedValue = round($item[\'unit_value\'] * $item[\'quantity\'], 2);',
    $content
);
$content = str_replace(
    'abs($item[\'total_value\'] - $expectedTotal)',
    'abs($item[\'total_value\'] - $expectedValue)',
    $content
);
$content = str_replace(
    '(esperado: {$expectedTotal})',
    '(esperado: {$expectedValue})',
    $content
);
$content = str_replace(
    '$calcTotal += $item[\'total_value\'];',
    '$calcTotal += $item[\'total_value\'] + ($item[\'total_igv\'] ?? 0);',
    $content
);
file_put_contents('app/Services/Signia/Engines/QpseEngine.php', $content);
echo "DONE";
